### ISSUE TITLE
'Server Side Processing' needs more advice

### ISSUE THREAD

**Issue by ToonTalk** [ox-it/LiveData_htmlwidgets#51]
I think many readers will wonder how to convert their spreadsheet into a JSONP server. Need a few sentences and links about this.

Also https://datatables.net/examples/server_side/jsonp.html says 

> JSONP is one of several methods to allow the use of JSON data from any server

**Comment by charliejhadley** [comment]
Agreed - add a "find out more here" link to somewhere relative permanent.
